src/openms/include/OpenMS/FORMAT/ParamCWLFile.h

```diff
@@ -13,60 +13,37 @@ namespace OpenMS
 {
 
   /**
-  @brief Load from JSON (in a Common Workflow Language (CWL) compatible way) into the Param class.
+  @brief Exports .cwl files.
 
-        The JSON file must contain one top level mapping of param value names to actual values.
-        These values can be one of the following types:
-            - null
-            - boolean
-            - int
-            - long
-            - float
-            - double
-            - string
-            - a CWL style file path ({ "class": "File", "path": "./myFolder/myFile.txt" })
-            - an array of these
-
-        param value names match the command line option without the leading '-'. Optionally the ':'
-        can be replaced with a double underscore "__".
-@code
-{
-    "in": {
-        "class": "File",
-        "path": "./myFolder/myFile.txt"
-    },
-    "out_prefix": "test_cwl_",
-    "algorithm:threshold": 5,
-    "algorithm:score_type": "ID"
-}
-@endcode
-
-Same file with "__" instead of ':' as the section separator.
-@code
-{
-    "in": {
-        "class": "File",
-        "path": "./myFolder/myFile.txt"
-    },
-    "out_prefix": "test_cwl_",
-    "algorithm__threshold": 5,
-    "algorithm__score_type": "ID"
-}
-@endcode
-*/
+        If Names include ':' it will be replaced with "__";
+  */
```

Comment:
make clear what is the difference between ParamCWL vs ParamJSON and crosslink in doxygen.